dts/msm8226/msm8226-samsung-ms01.dts

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id =  <0x91 0x08 0x10001 0x9e 0x08 0x10001 0x9f 0x08 0x10001 0xc6 0x08 0x10001 0xcd 0x08 0x10001>;
```

Comment:
```suggestion
	qcom,msm-id =  <0x91 0x08 0x10001>, <0x9e 0x08 0x10001>, <0x9f 0x08 0x10001>, <0xc6 0x08 0x10001>, <0xcd 0x08 0x10001>;
```

but this looks like MTP which is not what samsung usually uses. Are you sure you need this file at all?

Reply:
I am not sure how could i fix

Reply:
drop the file and check if lk2nd still boots on the device?

Reply:
Well it for sure will , but i thought those are 2 different revisions

Reply:
Unless you can confirm that there is a device that really needs to match mtp to work, please drop. This really is not something samsung ever used in my experience, and is just from the default qcom code most likely

Reply:
done